vivisect/analysis/i386/calling.py

```diff
@@ -88,6 +92,16 @@ def buildFunctionApi(vw, fva, emu, emumon):
 
         vw.setFunctionMeta(fva, "UndefRegUse", undefkeys)
 
+    # if we're callee cleanup, make sure we *actually* clean up our space
+    # otherwise, revert us to caller cleanup
+    if emumon.endstack:
+        stkoff = int((emumon.endstack - stkstart) / 4)
+        if callconv in argnames:
+            # do our stack args line up with what we cleaned up?
+            if abs(stkoff) != stackargs:
+                # we're probably caller cleanup then
+                callconv = callconv + 'er'
```

Comment:
Exactly. 

I'm not particularly beholden to the names, but it is effective. And I wanted something relatively terse since 'fastcall_callercleanup' seemed a bit too excessive. I'm open to name suggesstion since the "-er" addition is a bit kludge-y.

Reply:
Hmm. Trying for a more generic name, how about fastcall_caller? Little repetitive for my tastes, but I don't want to have to be specific about every possible compiler setting/name, and fastcall_caller is a little more specific.